lib/components/Timeline/Timeline.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { Section } from '../../layouts/Section/Section'
+import { Timeline, TimelineEventProps } from './Timeline'
+
+const meta: Meta<typeof Timeline> = {
+  title: 'Components/Timeline',
+  component: Timeline,
+  tags: ['autodocs'],
+  parameters: {
+    controls: {
+      sort: 'requiredFirst',
+    },
+  },
+}
+
+export default meta
+type Story = StoryObj<typeof Timeline>
+
+const sampleEvents: TimelineEventProps[] = [
+  {
+    date: '3rd January 2020',
+    title: 'What Is Lorem Ipsum?',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.",
+  },
+  {
+    date: '21st June 2019',
+    title: 'What Is Lorem Ipsum?',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+  },
+  {
+    date: '15th April 2018',
+    title: 'What Is Lorem Ipsum?',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+  },
+  {
+    date: '22nd March 2017',
+    title: 'What Is Lorem Ipsum?',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard.",
+  },
+]
```

Comment:
Let's move this to the /lib/data folder where we have all the other data. We sometimes import those files in other story examples

Reply:
Fixed, created TimelineData file in lib/data folder